src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,10 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
-        {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-        }
-
         [Fact, PlatformSpecific(PlatformID.Linux)]
         public void CheckLinux()
         {
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
Thanks! Will do.
I am sorry I was using cloud-based editor for GitHub: https://ide.c9.io/ (C9: all you need is a web browser) and I heavily rely on CI. (everything :cloud:-based spirit). Unlike AppVeyor / TravisCI, Jenkins CI doesn't cover forks by default, so I had to open the PR before verifying the changes. :smile: